ITEM.tsv

```diff
@@ -3037,16 +3037,16 @@ ITEM_20150317_003036	Treasure that belonged to human victims and was hoarded by
 ITEM_20150317_003037	Loose Package 
 ITEM_20150317_003038	Broken Necklace from Thorn Forest 
 ITEM_20150317_003039	Bushobell's Bundle 
-ITEM_20150317_003040	Soldier's Luggage  
-ITEM_20150317_003041	Belongings from a soldier, stolen by a Leafbug. Looks like it's been stolen many times since then. 
-ITEM_20150317_003042	Piece of Vubbe Cloth  
-ITEM_20150317_003043	A ripped cloth piece from Vubbe. It's from the dirty waist area. 
+ITEM_20150317_003040	$Soldier's Luggage  
+ITEM_20150317_003041	$Belongings from a soldier, stolen by a Leafbug. Looks like it's been stolen many times since then. 
+ITEM_20150317_003042	$Piece of Vubbe Cloth  
+ITEM_20150317_003043	$A ripped cloth piece from Vubbe. It's from the dirty waist area. 
 ITEM_20150317_003044	Weaver Claw 
 ITEM_20150317_003045	{memo X}If you gather enough of these, they may have a use. 
-ITEM_20150317_003046	Relief Goods 
-ITEM_20150317_003047	Goods used as a provision for emergencies in the Mining Town. 
-ITEM_20150317_003048	Reinforcement Material 
-ITEM_20150317_003049	Material required for reinforcement of the mine's exterior wall. Collect them before the wall crumbles. 
+ITEM_20150317_003046	$Relief Goods 
+ITEM_20150317_003047	$Goods used as a provision for emergencies in Miners' Village. 
+ITEM_20150317_003048	$Reinforcement Stone 
```

Comment:
Yep it does, I did this quest a lot in iCBT so I know it's a stone (more like boulder) that you gather.